client/app/scripts/superdesk-archive/module.js

```diff
@@ -217,7 +217,7 @@ define([
          * and Killed.
          *
          * @param {Object} item
-         * @return true if the state of the item is in one of the published states, false otherwise.
+         * @return boolean if the state of the item is in one of the published states, false otherwise.
```

Comment:
Hm, that's a bit strange... could it be that PyCharm's settings need to be adjusted?
(just for the reference, here's the [ngDocs documentation](https://github.com/idanush/ngdocs/wiki/API-Docs-Syntax#methods) for the method annotation syntax)

In any case, it's fine for now, especially since there are some minor inconsistencies in the docstrings at other places, too, and a more holistic approach would be needed anyway.